docs/src/use/suppressions.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Bulk Suppressions
+eleventyNavigation:
+    key: suppressions
+    parent: use eslint
+    title: Bulk Suppressions
+    order: 8
+
+---
+
+Enabling a new lint rule as `error` can be counterintuitive when the codebase has many violations and the rule isn't auto-fixable. A good example is [`@typescript-eslint/no-explicit-any`](https://typescript-eslint.io/rules/no-explicit-any/). Unless the rule is enabled during the early stages of the project, it becomes harder and harder to enable it as the codebase grows. Existing violations must be resolved before enabling the rule, but while doing that other violations might creep in.
+
+To address this, ESLint provides a way to suppress existing violations for one or more rules. While the rule will be enforced for new code, the existing violations will not be reported. This way, you can address the existing violations at your own pace.
+
+Please note that this feature is only available for rules that are enabled as `error`. If a rule is enabled as `warn`, ESLint will not suppress the violations.
+
+## Suppressing Violations
+
+After you enable a rule as `error`, you can suppress all the existing violations at once by using the `--suppress-all` flag. It is recommended to execute the command with the `--fix` flag first so that you don't suppress violations that can be auto-fixed.
```

Comment:
```suggestion
After you enable a rule as `"error"` in your configuration file, you can suppress all the existing violations at once by using the `--suppress-all` flag. It is recommended to execute the command with the `--fix` flag so that you don't suppress violations that can be auto-fixed.
```